.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: php
+php:
+    - 7.1
+    - 7.2
+
+branches:
+    only:
+        - master
+
+install:
+    - composer install --prefer-source
```

Comment:
why prefer-source ?

Reply:
Copy/paste 👼 I'll change to prefer-dist which is better here ofc.